src/textual/drivers/_input_reader.py

```diff
@@ -1,10 +1,11 @@
 import sys
+from typing import Final
 
 __all__ = ["InputReader"]
 
-WINDOWS = sys.platform == "win32"
+WINDOWS: Final = sys.platform == "win32"
 
-if WINDOWS:
+if sys.platform == "win32":
```

Comment:
If we are going to do this, do we need the `WINDOWS` constant?

Reply:
I suspect the WINDOWS constant is redundant

Reply:
Suggest we remove it.